src/utils/reshaped_reflection.fs

```diff
@@ -0,0 +1,362 @@
+// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.FSharp.Core
+
+//Replacement for: System.Security.SecurityElement.Escape(line) All platforms
+module internal XmlAdapters =
+    open System.Text
+    open Microsoft.FSharp.Collections
+
+    let s_escapeChars = [| '<'; '>'; '\"'; '\''; '&' |]
+
+    let getEscapeSequence c =
+        match c with
+        | '<'  -> "&lt;"
+        | '>'  -> "&gt;"
+        | '\"' -> "&quot;"
+        | '\'' -> "&apos;"
+        | '&'  -> "&amp;"
+        | _ as ch -> ch.ToString()
+
+    let escape str =
```

Comment:
This can be simplified to just `let escape str = String.collect getEscapeSequence str`